CHANGELOG.md

```diff
@@ -21,6 +21,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - separated `snforge` to subcommands: `snforge test`, `snforge init` and `snforge clean-cache`. 
 Read more [here](https://foundry-rs.github.io/starknet-foundry/appendix/forge.html).
 - `starknet::get_block_info` now returns correct block info in a forked block
+- forking `Pending` block has become disallowed
```

Comment:
Should we add this to the `Breaking` section?

Reply:
It is not automated, we can consider adding it @cptartur

Reply:
We haven't been explicit about breaking changes in the past, we can consider it to simplify migrations for users.